configuration/wireless/access-point-routed.md

```diff
@@ -0,0 +1,190 @@
+                                     |   | 192.168.4.1 |          | 192.168.4.2 |
+                                     |   +-------------+          +-------------+
+                 +- Router ----+     |
+                 | Firewall    |     |   +- PC#2 ------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 10.10.0.3   |
+                 |   10.10.0.1 |     |   +-------------+
+                 +-------------+     |
+                                     |   +- PC#1 ------+
+                                     +---+ 10.10.0.4   |
+                                         +-------------+
+```
+This can be done using the inbuilt wireless features of the Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero W, or by using a suitable USB wireless dongle that supports access point mode.
+It is possible that some USB dongles may need slight changes to their settings. If you are having trouble with a USB wireless dongle, please check the forums.
+
+This documentation was tested on a Raspberry Pi 3B running a factory installation of Raspbian Buster Lite (July 2019). 
+
+<a name="intro"></a>
+## Before you start
+
+* Ensure you have administrative access to your Raspberry Pi. The network setup will be modified as part of the installation: local access, with screen and keyboard connected to your Raspberry Pi, is recommended.
+* Connect your Raspberry Pi to the ethernet network and boot the Raspbian OS.
+* Ensure the Raspbian OS on your Raspberry Pi is [up to date](../../raspbian/updating.md) and reboot if packages were installed in the process.
+* Take note of the IP configuration of the ethernet network the Raspberry Pi is connected to: 
+    * In this document, we assume IP network `10.10.0.0/24` is configured on the ethernet LAN, and the Raspberry Pi is to manage IP network `192.168.4.0/24` for wireless clients.
+    * Please select another IP network for wireless, e.g. `192.168.10.0/24`, in case IP network `192.168.4.0/24` is already in use by your ethernet LAN.
+* Have a wireless client (laptop, smartphone, ...) ready to test your new access point.
+
+<a name="software-install"></a>
+## Install the access point and network management software
+
+In order to work as an access point, the Raspberry Pi needs to have the `hostapd` access point software package installed:
+
+```
+sudo apt install hostapd
+```
+Enable the wireless access point service to start when your Raspberry Pi boots:
+
+```
+sudo systemctl unmask hostapd
+sudo systemctl enable hostapd
+```
+
+In order to provide network management services (DNS, DHCP) to wireless clients, the Raspberry Pi needs to have the `dnsmasq` software package installed:
+
+```
+sudo apt install dnsmasq
+```
+Finally, install `netfilter-persistent` and its plugin `iptables-persistent`. This utilty helps saving firewall rules and restores them when the Raspberry Pi boots:
+
+```
+sudo DEBIAN_FRONTEND=noninteractive apt install -y netfilter-persistent iptables-persistent
```

Comment:
Is `DEBIAN_FRONTEND=noninteractive` really necessary? (if it is, maybe it's worth explaining why?)

Reply:
Boring explanation: The install process interactively asks if you want to save the current iptables state. "noninteractive" + option "-y" skips the question and answers yes, making copy-paste possible.

To me it is a pure technicality, I wouldn't mind not explaining the unusual options used here.